client/src/components/Invites.js

```diff
@@ -46,19 +121,57 @@ const Invites = (props) => {
     setShowRequests(false);
   };
 
+  useEffect(() => {
+    //axios fetch all the PENDING invites user has RECEIVED
+    (async function fetchPendingInvites() {
+      const res = await axios.get(`user/${testUserId}/invitations/in/pending`);
+      const pendingInvites = res.data;
+      setInvitesIn(pendingInvites);
```

Comment:
This route can return a 200, a 204, a 404, or a 500, so we want to check what the response is before we assume that the data is good.

Since we also need the outgoing invitations on the page, would it make sense to have a single route that returns both?